Makefile.am

```diff
@@ -302,7 +305,7 @@ libini_config_la_LIBADD = \
     libpath_utils.la \
     libref_array.la \
     libbasicobjects.la
-libini_config_la_LDFLAGS = -liconv -lintl \
+libini_config_la_LDFLAGS = $(LIBICONV) $(LIBINTL)\
```

Comment:
* please add library related options to `*_LIBADD` instead of `*_LDFLAGS`
* please use `LTLIBINTL` instead of `LIBINTL`. It works better for me on freeBSD
* there is also the same issue with libpath_utils on freeBSD

```
root@:~/ding-libs # make check
Making check in .
make  path_utils_ut  ini_augment_ut_check  ini_configmod_ut_check  ini_parse_ut_check  ini_validators_ut_check dhash_test  dhash_example dhash_ut_check collection_ut  collection_stack_ut collection_queue_ut  ref_array_ut simplebuffer_ut  ini_config_ut ini_comment_ut  ini_valueobj_ut ini_parse_ut  ini_augment_ut ini_configmod_ut  ini_save_ut
/bin/sh ./libtool  --tag=CC    --mode=link cc -Wall -Wshadow -Wstrict-prototypes -Wpointer-arith -Wcast-qual  -Wcast-align -Wwrite-strings -Wextra -Wno-unused-parameter  -Wno-sign-compare -Wformat-security -Wmissing-prototypes -Wundef  -I/usr/local/include -D_THREAD_SAFE -pthread  -g -O2   -o path_utils_ut path_utils/path_utils_ut-path_utils_ut.o -L/usr/local/lib -lcheck  libpath_utils.la 
libtool: link: cc -Wall -Wshadow -Wstrict-prototypes -Wpointer-arith -Wcast-qual -Wcast-align -Wwrite-strings -Wextra -Wno-unused-parameter -Wno-sign-compare -Wformat-security -Wmissing-prototypes -Wundef -I/usr/local/include -D_THREAD_SAFE -pthread -g -O2 -o .libs/path_utils_ut path_utils/path_utils_ut-path_utils_ut.o  -L/usr/local/lib -lcheck ./.libs/libpath_utils.so -pthread -Wl,-rpath -Wl,/usr/local/lib
./.libs/libpath_utils.so: undefined reference to `libintl_gettext'
cc: error: linker command failed with exit code 1 (use -v to see invocation)
*** Error code 1

Stop.
make[2]: stopped in /root/ding-libs
*** Error code 1

Stop.
make[1]: stopped in /root/ding-libs
*** Error code 1

Stop.
make: stopped in /root/ding-libs
```

and objdump confirms that for me:
```
root@:~/ding-libs # objdump -p ./.libs/libini_config.so | grep NEEDED
  NEEDED      libintl.so.8
  NEEDED      libcollection.so.4
  NEEDED      libpath_utils.so.1
  NEEDED      libref_array.so.1
  NEEDED      libbasicobjects.so.0
  NEEDED      libc.so.7
root@:~/ding-libs # objdump -p ./.libs/libpath_utils.so | grep NEEDED
  NEEDED      libc.so.7
root@:~/ding-libs # objdump -T ./.libs/libpath_utils.so | grep libintl_gettext
0000000000000000      D  *UND*  0000000000000000              libintl_gettext
root@:~/ding-libs # objdump -T ./.libs/libini_config.so | grep libintl_gettext
0000000000000000      DF *UND*  0000000000000017              libintl_gettext
```

Reply:
Done. I have no access to a FreeBSD machine, and there's still no CI here, so no idea if the changes work. If they don't, would you mind figuring out how to get it working on FreeBSD?